pkg/controller/perconaservermongodb/mgo.go

```diff
@@ -199,14 +198,14 @@ func (r *ReconcilePerconaServerMongoDB) reconcileCluster(cr *api.PerconaServerMo
 	return clusterInit, nil
 }
 
-func inShard(client *mgo.Client, podName string) (bool, error) {
+func inShard(client *mgo.Client, rsName string) (bool, error) {
 	shardList, err := mongo.ListShard(context.TODO(), client)
```

Comment:
can we use `getShardMap` instead?
I want to be 100% sure that all Pods are added to Sharding

Reply:
We do it here 
https://github.com/percona/percona-server-mongodb-operator/blob/main/pkg/controller/perconaservermongodb/mgo.go#L127-L176

In any case we did not check all the pods before, only the first one.
https://github.com/percona/percona-server-mongodb-operator/blob/main/pkg/controller/perconaservermongodb/mgo.go#L101